catalogue/webapp/pages/search/index.tsx

```diff
@@ -31,16 +85,119 @@ export const getServerSideProps: GetServerSideProps<Props | AppErrorProps> =
       return { notFound: true };
     }
 
-    return {
-      props: removeUndefinedProps({
-        serverData,
-        // TODO Harrison to explore what properties we'd want here
-        pageview: {
-          name: 'search',
-          properties: {},
-        },
-      }),
-    };
+    const query = context.query;
+    const params = fromQuery(query);
+
+    const defaultProps = removeUndefinedProps({
+      serverData,
+      works: undefined,
+      images: undefined,
+      stories: undefined,
+      // TODO Harrison to explore what properties we'd want here
+      pageview: {
+        name: 'stories',
+        properties: {},
+      },
+    });
+
+    // Stop here if no query has been entered
+    if (!params.query) {
+      return {
+        props: defaultProps,
+      };
+    }
+
+    try {
+      /*
+       * Stories
+       */
+      const storiesFetch = await getStories({
+        query,
+        pageSize: 3,
+      });
+
+      const stories =
+        storiesFetch && storiesFetch.type !== 'Error'
+          ? storiesFetch.results
+          : undefined;
+
+      // An error shouldn't stop the other results from displaying
+      if (storiesFetch.type === 'Error') {
+        console.error('Error fetching stories:', storiesFetch.label);
+      }
+
+      /*
+       * Works
+       */
+      const worksAggregations = [
+        'workType',
+        'availabilities',
+        'genres.label',
+        'languages',
+        'subjects.label',
+        'contributors.agent.label',
+      ];
```

Comment:
Why do we want work aggregations? They're a moderately expensive part of the query; would we ever render them on the overview page?

Reply:
I think based on our conversation from these morning, these can effectively go 🙌